server/schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE users (
+  id                    SERIAL PRIMARY KEY,
+  firstname             VARCHAR(30) NOT NULL,
+  lastname              VARCHAR(30) NOT NULL,
+  email                 VARCHAR(30) NOT NULL,
+  password              VARCHAR(12) NOT NULL,
+  createdAt             TIMESTAMP  NOT NULL
+);
+
+CREATE TABLE stars (
+    id                  SERIAL PRIMARY KEY,
+    user_id             INTEGER,
+    competence_id       INTEGER NOT NULL,
+    title               TEXT NOT NULL,
+    situation           TEXT NOT NULL,
+    task                TEXT NOT NULL,
+    action              TEXT NOT NULL,
+    result              TEXT NOT NULL,
+    createdAt           TIMESTAMP  NOT NULL,
+    UpdatedAt           TIMESTAMP  NOT NULL
```

Comment:
+1 for this. This is really good practice, adding date fields is always a great thing to do. Nice work!

Reply:
Thank you. Learnt that from Elmira